service/controller/v1/resource/app/resource.go

```diff
@@ -66,23 +66,14 @@ func (r *Resource) Name() string {
 	return Name
 }
 
-func containsAppCRs(list []*applicationv1alpha1.App, item *applicationv1alpha1.App) bool {
-	_, err := getAppCRByName(list, item.Name)
-	if err != nil {
-		return false
-	}
-
-	return true
-}
-
-func getAppCRByName(list []*applicationv1alpha1.App, name string) (*applicationv1alpha1.App, error) {
+func getAppCR(list []*applicationv1alpha1.App, namespace, name string) (*applicationv1alpha1.App, bool) {
```

Comment:
There is a new `namespace` argument but it is never used for comparison.

Reply:
LoL